src/c_lib.rs

```diff
@@ -589,6 +589,35 @@ pub extern "C" fn execute_vade(
                 Ok("".to_string())
             }
         }),
+        #[cfg(feature = "plugin-vc-zkp-bbs")]
+        "helper_create_self_issued_credential" => runtime.block_on({
+            async {
+                let mut vade_evan = get_vade_evan(
+                    Some(&str_config),
+                    #[cfg(all(feature = "target-c-lib", feature = "capability-sdk"))]
+                        ptr_request_list,
+                    #[cfg(all(feature = "target-c-lib", feature = "capability-sdk"))]
+                        request_function_callback,
+                )
+                    .map_err(stringify_generic_error)?;
+                let use_valid_until = match arguments_vec.get(1) {
+                    Some(value) => value.to_lowercase() == "true",
+                    None => false,
+                };
```

Comment:
Can we drop this with the valid until as an extra parameter? ^^

```suggestion
```

Reply:
Sure, thanks, leftover...